source-airtable/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM python:3.11-slim as builder
```

Comment:
I think we need to find a way to have this be a template execution, collapsed with `Dockerfile.example` from this commit, with an easy garden path so that devs can largely ignore Docker unless they need custom packages, etc.

Copy / paste of Dockerfiles will get unwieldy pretty much immediately (it's already unwieldy for our technical connectors in this repo), and while it's not in scope, they _also_ could benefit from establishing a better precedent here.

We should probably pick and try to standardize on a tool like https://github.com/hairyhenderson/gomplate for executing a custom template given a JSON input over the CLI (this is literally just the first Google search match and looked reasonable, but I have no strong opinion on the particular template renderer we use).

Reply:
As discussed over Slack, another option is to use relative symbolic links -- tracked by git -- paired with `--build-arg`'s that pass in (minimally) the connector name.

`--build-args` can also read from the environment so there may be an opportunity to selectively use .env files to override ARGs of a common Dockerfile ?

Reply:
It turns out that you can just specify which Dockerfile to use with `docker build -f <path/to/Dockerfile>`, so I updated the Dockerfile template to use build args, and `build-local.sh` to provide them.